py/desispec/database/redshift.py

```diff
@@ -474,7 +474,7 @@ def load_file(filepath, tcls, hdu=1, expand=None, convert=None, index=None,
     return
 
 
-def load_zcat(datapath=None, q3c=False):
+def load_zbest(datapath=None, q3c=False):
```

Comment:
I'm confused about the DB loading roadmap here.  This function docstring says that it is deprecated since there should now be a single redshift catalog file, but I don't see where the code is for loading that file.  Previously we had specifically moved away from loading N>>1 individual zbest files (e.g. to avoid the hardcoded "spectra-64/*/*/zbest-64*.fits assumptions).  Where is the target catalog loading now done (for when the target catalog exists)?

To be clear: DB loading should match the data available from "real" processing like the minitest notebook.  If it is also able to support the subset of files available from the desispec integration test, great.  If not, we shouldn't go to great lengths to alter the DB loading to match a non-standard case.

Reply:
Assuming the zcatalog file *exists*, it is loaded by `desispec.database.redshift.load_file()`, just like the target, truth and exposure files.  If it does not exist, then a subset of the data can be loaded from the zbest files, and that's what I've updated.  I can remove that deprecation warning if necessary.

However, it sounds like you want the integration test to actually create the zcatalog file.  Is it even possible to do that without targeting data?  If so, we can try that.